baseplate/server.go

```diff
@@ -0,0 +1,70 @@
+package baseplate
+
+import (
+	"errors"
+	"io/ioutil"
+	"time"
+
+	"github.com/reddit/baseplate.go/log"
+	"gopkg.in/yaml.v2"
```

Comment:
Style nit: Please group imports into 3 groups (separated by blank lines): stdlibs, third-party imports (anything that's not stdlib and don't have `github.com/reddit/baseplate.go` prefix), and packages from this module (anything with `github.com/reddit/baseplate.go` prefix).